sssom/rdf_util.py

```diff
@@ -39,7 +40,7 @@ def expand_curie(curie: str) -> URIRef:
                 src, tgt = m.subject_id, m.object_id
             else:
                 src, tgt = m.object_id, m.subject_id
-            if not isinstance(src, EntityId) or not isinstance(tgt, EntityId):
+            if not isinstance(src, URIorCURIE) or not isinstance(tgt, URIorCURIE):
```

Comment:
Can we double check that this is correct? I thought we got rid of the URIorCURIE construct at some point. Is there a test that at least touches this method?

Reply:
`test_rewire.py` hits this code and all values of `tat` and `src` are of `<class 'sssom.sssom_datamodel.EntityReference'>`. I take out this check.